cmd/auth-provider-gcp/apis/credentialprovider/v1alpha1/BUILD

```diff
@@ -0,0 +1,37 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "doc.go",
+        "register.go",
+        "types.go",
+        "zz_generated.conversion.go",
+        "zz_generated.deepcopy.go",
+        "zz_generated.defaults.go",
+    ],
+    importmap = "k8s.io/kubernetes/vendor/k8s.io/kubelet/pkg/apis/credentialprovider/v1alpha1",
+    importpath = "k8s.io/kubelet/pkg/apis/credentialprovider/v1alpha1",
```

Comment:
Can we move this to cloud-provider-gcp/pkg/apis/credentialprovider/v1alpha1 and fix our references? Worries that a vendor update will accidentally pick up the actual k8s.io/kubelet/pkg/apis/credentialprovider/v1alpha1 stuff and cause some very strange errors.

Reply:
Sure thing, fixed